README.md

```diff
@@ -2,6 +2,7 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+![screenshot](src/bo.png)
 ## Available Scripts
 
 In the project directory, you can run:
```

Comment:
[OPTIONAL]

- Please include the live version of your application in your README.